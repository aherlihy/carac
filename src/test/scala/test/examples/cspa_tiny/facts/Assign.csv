Int	Int
73	74
46	81
46	82
50	85
50	86
40	87
40	88
89	90
42	91
42	92
52	93
52	94
57	95
57	96
99	39
78	114
115	53
123	89
134	136
161	108
171	69
172	72
173	76
180	170
191	64
192	62
193	67
201	202
190	203
157	49
210	211
212	43
54	197
228	157
203	56
234	184
211	41
244	175
259	49
261	39
263	45
265	56
268	41
270	51
274	275
199	231
276	255
288	195
315	98
306	321
295	51
