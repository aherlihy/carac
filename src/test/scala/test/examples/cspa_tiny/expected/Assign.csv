Int	Int
263	45
40	88
201	202
157	49
306	321
40	87
210	211
199	231
265	56
211	41
259	49
99	39
50	85
54	197
288	195
270	51
234	184
228	157
115	53
193	67
78	114
180	170
172	72
57	96
89	90
212	43
192	62
134	136
52	94
161	108
268	41
190	203
173	76
191	64
203	56
244	175
57	95
73	74
42	92
274	275
261	39
171	69
276	255
42	91
46	81
295	51
50	86
123	89
46	82
315	98
52	93
